code/__DEFINES/armor.dm

```diff
@@ -0,0 +1,11 @@
+
+#define ARMOUR_DROPOFF_START 50
+#define ARMOUR_MAXIMUM 75
+
+#define STANDARDISE_ARMOUR(val) round(CALCULATE_ARMOUR(val, ARMOUR_DROPOFF_START, (ARMOUR_MAXIMUM - ARMOUR_DROPOFF_START)), 1)
```

Comment:
```suggestion
#define STANDARDISE_ARMORR(val) round(CALCULATE_ARMOUR(val, ARMOUR_DROPOFF_START, (ARMOUR_MAXIMUM - ARMOUR_DROPOFF_START)), 1)
```
typo

Reply:
![image](https://github.com/BeeStation/BeeStation-Hornet/assets/26465327/5640c403-1f41-4643-8521-dec38b0431c3)
You're suggestion is the one with the typo.